prac_07/project_managment.py

```diff
@@ -18,16 +18,16 @@
 
 def main():
     """Handle projects"""
+    projects = []
```

Comment:
You should additionally load the projects from a default file automatically before the MENU is printed, and save it automatically after the user quits